custom/icds/templates/icds/messaging/indicators/kan/aww_vhnd_survey.txt

```diff
@@ -0,0 +1 @@
+As per the record, VHSND could not happen at your centre in the previous month. Please ensure this is conducted on regular basis every month.
```

Comment:
Looks like this one was missed. Does that matter?

Reply:
@millerdev This was left out intentionally because we haven't received translations for this message yet. I aim to update the translations for this specific message by the end of this week.